src/components/postgresConnect.ts

```diff
@@ -682,11 +682,17 @@ class PostgresDB {
     }
   }
 
+  // TODO: Rename to getReviewStageAndLevel
```

Comment:
I'd suggest moving this to the Action's databaseMethods file, then you can make the name change.

The general rule (since I've been caught out a couple of times) is that any methods in `postGresConnect` should be assumed to be used elsewhere, so we should be very cautious about editing them. And if they're exclusive to a particular Action, then they should stay in the Action package (databaseMethods), then we know they can safely be edited without affecting anything else.

Reply:
In this case should I move `getCurrentStageHistory` to incrementStage since it's only used in there?

Reply:
I don't know because how do you know there is one already if we keep spreading things on this separated files. I like having everything in the same file to allow for reusability

Reply:
Don't worry about moving this one specifically. We're slowly migrating them over where appropriate.

> I like having everything in the same file to allow for reusability

Yes, same, but it turned out that hardly any of these were re-usable since they're so specific to their use case. So the postGresConnect file was just becoming a bloated mess and very hard to work with. And the big danger is that it's really tricky to edit an existing query in there as it might mess up another Action that uses it. (This happened a couple of times, which is why we went in this direction). If we find we need a query that already exists for another Action, then it's no trouble to move it back to the shared one. But it makes life sooo much easier when working on actions not constantly scrolling and searching through a giant file.

Reply:
> I don't know because how do you know there is one already if we keep spreading things on this separated files. I like having everything in the same file to allow for reusability

I don't think that will happen because it's not very common to go looking on other actions when you are developing a new one. But, you know, it really doesn't matter. Is just a preference in this case. It's nicer to have thing that are related closer or at least easy to find.